src/app/modules/page-header/is-new-indicator/is-new-indicator.component.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { FeedbackDialogComponent } from 'app/modules/ix-feedback/feedback-dialog/feedback-dialog.component';
+
+@Component({
+  selector: 'ix-is-new-indicator',
```

Comment:
I think `ix-new-page-badge` is a better selector.

Reply:
> I think `ix-new-page-badge` is a better selector.

nice catch, thanks.